docs/sources/slack/example-api-responses/sanitized/discovery-conversations-info.json

```diff
@@ -8,7 +8,7 @@
       "is_archived":false,
       "is_general":false,
       "unlinked":0,
-      "creator":"{\"hash\":\"MOMSVXXFHYCrH2e8Zrt_0G9lcd_GK4cUKRXBmO2QAa0\"}",
+      "creator":"MOMSVXXFHYCrH2e8Zrt_0G9lcd_GK4cUKRXBmO2QAa0",
```

Comment:
why is the hash / json gone? I don't find the change that makes that happen... shouldn't at least be t~HASH ??

Reply:
it's just the format change; default is now `URL_SAFE_TOKEN` instead of `JSON`

Reply:
but yeah, i agree should have `t~` for claritify - I've found a path through which that wasn't being added

Reply:
seems like not anything introduced here; but path that was never hit bc we used JSON format by default in most places.